environment.yml

```diff
@@ -0,0 +1,8 @@
+name: tutorial_short
+channels:
+  - conda-forge
+  - defaults
```

Comment:
safe to remove `defaults` since all them packages are on `conda-forge` (safer not to mix channels); isn't `gxx_linux-64` an actual dependency of `ruby`?

Reply:
```suggestion
```